dvc/tree/webdav.py

```diff
@@ -81,12 +81,6 @@ def __init__(self, repo, config):
             if self.user is None and self.path_info.user is not None:
                 self.user = self.path_info.user
 
-            # If username specified add to path_info
-            if self.user is not None:
-                self.path_info.user = self.user
-        else:
-            self.path_info = None
-
```

Comment:
It's not necessary to add user to the url, as we also provide the login config anyway to the webdav Client.